# GION

Origin Notes: The data was originally collected for a Bioinformatics paper, “https://doi.org/10.1186/1471-2105-11-378”. It was then used in “https://link.springer.com/chapter/10.1007/978-3-662-45803-7_10” as a benchmark for their layout and experiment. However, the links to the original data no longer work.
graph features handled: dense, large, sparse
Graph features in papers: generic,large
Origin Paper: GION: Interactively Untangling Large Graphs on Wall-Sized Displays (https://www.notion.so/GION-Interactively-Untangling-Large-Graphs-on-Wall-Sized-Displays-2992d3da3efa4626b5a0384500f54a4e?pvs=21), Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data (https://www.notion.so/Graph-based-clustering-and-characterization-of-repetitive-sequences-in-next-generation-sequencing-da-e3dd4bdceaec4781b3d0c70693df40e0?pvs=21)
Originally found at: http://wcl.ml.unisa.edu.au/graph-untangling/graphs.zip
Size: 8 graphs, 1159-7885 nodes, 6424-427406 edges
Number of Graphs: 8
Appeared in years: 2017,2014
Type of Collection: Lost/Unavailable
is it stored properly?: No
must be analyzed: Yes
In repo?: No
Related to Literature - Algorithm (1) (Dataset tag relations): GION: Interactively Untangling Large Graphs on Wall-Sized Displays (https://www.notion.so/GION-Interactively-Untangling-Large-Graphs-on-Wall-Sized-Displays-875925d42ff445e38ffef123e4ce771c?pvs=21), Drawing Big Graphs Using Spectral
Sparsification (https://www.notion.so/Drawing-Big-Graphs-Using-Spectral-Sparsification-89da2043d0444f75a6c663b1fc999512?pvs=21)
cleaned format?: No
duplicate?: No
link works?: No
Added in paper: Yes
Origin paper plaintext: GION: Interactively Untangling Large Graphs on Wall-Sized Displays, Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data
Page id: fb235f160e504395a884f5a18fca6f17
unavailable/skip: Yes
Cleaned ALL data: No
first look: No
Related to Literature - Algorithm (Dataset tag relations) 1: Drawing Big Graphs Using Spectral
Sparsification (../Benchmark%20sets%200cc6b5e454304aec98f3b59b1a720476/Literature%20ad87f14e7097454fb2f784e2c8a2797a/Literature%20-%20Algorithm%2012e01bfc60a84007aa7d2d34293e123d/Drawing%20Big%20Graphs%20Using%20Spectral%20Sparsification%20e5d3efdca48541f2b1789ec74357ebf6.md), GION: Interactively Untangling Large Graphs on Wall-Sized Displays (../Benchmark%20sets%200cc6b5e454304aec98f3b59b1a720476/Literature%20ad87f14e7097454fb2f784e2c8a2797a/Literature%20-%20Algorithm%2012e01bfc60a84007aa7d2d34293e123d/GION%20Interactively%20Untangling%20Large%20Graphs%20on%20Wall%20774568ebdd114a8790c026bfa0b125f0.md)

# Body

### Descriptions from Literature

From “[Drawing Big Graphs Using Spectral Sparsification](https://doi.org/10.1007/978-3-319-73915-1_22)”:

> this consists of RNA sequence graphs that are used for the analysis of repetitive sequences in sequencing data; these graphs have been used in previous experiments. They are locally dense and globally sparse, and generally have distinctive shapes.
> 

From “[GION: Interactively Untangling Large Graphs on Wall-Sized Displays](https://link.springer.com/chapter/10.1007/978-3-662-45803-7_10)”:

> We selected graphs that come from a real world application where graph visualization is used for data analysis and the graph size and layout quality requirements pose a challenge for state-of-the-art layout methods. Our graph set consists of RNA sequence graphs that are used for the analysis of repetitive sequences in sequencing data [11].
> 

> Graph properties are provided in Table 1. The set of graphs can be downloaded from http://wcl.ml.unisa.edu.au/graph-untangling/graphs.zip. All graphs have high local density, and a sparse global structure that allows to create layouts far from hairballs that are showing the structure well.
> 

![Untitled](GION%20fb235f160e504395a884f5a18fca6f17/Untitled.png)

### Example Figures

From “[GION: Interactively Untangling Large Graphs on Wall-Sized Displays](https://link.springer.com/chapter/10.1007/978-3-662-45803-7_10)”:

![Untitled](GION%20fb235f160e504395a884f5a18fca6f17/Untitled%201.png)

From “[Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data](https://link.springer.com/article/10.1186/1471-2105-11-378)”:

![Untitled](GION%20fb235f160e504395a884f5a18fca6f17/Untitled%202.png)

**Fig. 4. Examples of graph layouts derived from clusters of repetitive sequences**. Graph layouts were calculated using the 3D version of Fruchterman and Reingold algorithm from which a 2D projection is shown. Individual reads are represented by vertices and similar reads are connected by edges. Individual clusters are described further in Table 1.

=== STOP RENDERING ===

**NOTE: eliminate the relationships between different tables**

![Untitled](GION%20fb235f160e504395a884f5a18fca6f17/Untitled%203.png)

From [GION: Interactively Untangling Large Graphs on Wall-Sized Displays](../Benchmark%20sets%200cc6b5e454304aec98f3b59b1a720476/Literature%20ad87f14e7097454fb2f784e2c8a2797a/Literature%20-%20Algorithm%2012e01bfc60a84007aa7d2d34293e123d/GION%20Interactively%20Untangling%20Large%20Graphs%20on%20Wall%20774568ebdd114a8790c026bfa0b125f0.md) 
Graph properties are provided in Table 1. The set of graphs can be downloaded from

http://wcl.ml.unisa.edu.au/graph-untangling/graphs.zip. All graphs have high local density, and a sparse global structure that allows to create layouts far from hairballs that are showing the structure well

Dead link! [http://wcl.ml.unisa.edu.au/graph-untangling/graphs.zip](http://wcl.ml.unisa.edu.au/graph-untangling/graphs.zip) ← Emailed [michael@20papercups.net](mailto:michael@20papercups.net) (seems to have left academia and this is the email under his personal website)